bindings/julia/Displaz.jl

```diff
@@ -50,15 +49,54 @@ function write_ply_points(fileName, nvertices, fields)
             for i = 1:size(value,2)
                 propName = ply_property_name(semantic, i)
                 write(fid, "property $typeName $propName\n")
-            end
+        	end
         end
         write(fid, "end_header\n")
         for (_,value) in convertedFields
-            write(fid, value')
+			write(fid, value')
         end
     end
 end
 
+function write_ply_lines(fileName, position, color)
+	vertexValid = ~all(isnan(position),2)
+	lineStarts = find(x -> x == 1, diff([false;vertexValid],1))
```

Comment:
The julia naming convention (which I try to follow, though I can see there's some slipups elsewhere) is to use `underscore_case`, or `squishedcase` for short names.  Can you change the variable names to follow that convention for consistency?